client/rpc/src/lib.rs

```diff
@@ -66,6 +67,52 @@ pub mod frontier_backend_client {
 	use fc_rpc_core::types::BlockNumber;
 	use fp_storage::{EthereumStorageSchema, PALLET_ETHEREUM_SCHEMA};
 
+	/// Implements a default runtime storage override.
+	/// It assumes that the balances and nonces are stored in pallet `system.account`, and
+	/// have `nonce: Index` = `u32` for  and `free: Balance` = `u128`.
+	pub struct DefaultEthereumRuntimeStorageOverride<M>(pub std::marker::PhantomData<M>);
```

Comment:
This default may or may not be super useful as we ourselves are using `AccountId20` while frontier is targetting `AccountId32`. The basic pallet system override might be trivial to copy paste from the template instead.